apis/Google.Cloud.Firestore.Data/Google.Cloud.Firestore.Data/Query.cs

```diff
@@ -33,13 +43,415 @@ public class Query
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Unlike LINQ's OrderBy method, this call adds additional subordinate orderings to any
+        /// additionally specified. So <c>query.OrderBy("foo").OrderBy("bar")</c> is similar
+        /// to a LINQ <c>query.OrderBy(x => x.Foo).ThenBy(x => x.Bar)</c>.
+        /// </para>
+        /// <para>
+        /// This method cannot be called after a start/end cursor has been specified with
+        /// <see cref="StartAt"/>, <see cref="StartAfter(object[])"/>, <see cref="EndAt(object[])"/> or <see cref="EndBefore(object[])"/>.
+        /// </para>
+        /// </remarks>
+        /// <param name="fieldPath">The dot-separated path to order by. Must not be null or empty.</param>
+        /// <returns>A new query based on the current one, but with the additional specified ordering applied.</returns>
+        public Query OrderBy(string fieldPath) => OrderBy(fieldPath, Direction.Ascending);
+
+        /// <summary>
+        /// Adds an additional descending ordering by the specified path.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Unlike LINQ's OrderBy method, this call adds additional subordinate orderings to any
+        /// additionally specified. So <c>query.OrderBy("foo").OrderByDescending("bar")</c> is similar
+        /// to a LINQ <c>query.OrderBy(x => x.Foo).ThenByDescending(x => x.Bar)</c>.
+        /// </para>
+        /// <para>
+        /// This method cannot be called after a start/end cursor has been specified with
+        /// <see cref="StartAt"/>, <see cref="StartAfter(object[])"/>, <see cref="EndAt(object[])"/> or <see cref="EndBefore(object[])"/>.
+        /// </para>
+        /// </remarks>
+        /// <param name="fieldPath">The dot-separated path to order by. Must not be null or empty.</param>
+        /// <returns>A new query based on the current one, but with the additional specified ordering applied.</returns>
+        public Query OrderByDescending(string fieldPath) => OrderBy(fieldPath, Direction.Descending);
+
+        /// <summary>
+        /// Adds an additional ascending ordering by the specified path.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Unlike LINQ's OrderBy method, this call adds additional subordinate orderings to any
+        /// additionally specified. So <c>query.OrderBy("foo").OrderBy("bar")</c> is similar
+        /// to a LINQ <c>query.OrderBy(x => x.Foo).ThenBy(x => x.Bar)</c>.
+        /// </para>
+        /// <para>
+        /// This method cannot be called after a start/end cursor has been specified with
+        /// <see cref="StartAt"/>, <see cref="StartAfter(object[])"/>, <see cref="EndAt(object[])"/> or <see cref="EndBefore(object[])"/>.
+        /// </para>
+        /// </remarks>
+        /// <param name="fieldPath">The dot-separated path to order by. Must not be null.</param>
```

Comment:
Is this param comment correct? Is a `FieldPath` dot-separated?

Reply:
Nope, c'n'p error. All the *string* parameters should say "dot-separator".